.github/workflows/sbml-semantic-test-suite.yml

```diff
@@ -5,8 +5,9 @@ on:
       - develop
       - master
   pull_request:
-    branches:
-      - master
+    paths:
+      - python/amici/sbml_import.py
+      - python/amici/ode_export.py
```

Comment:
add `scripts/run-SBMLTestsuite.sh` and `tests/testSBMLSuite.py`?

Reply:
Sounds reasonable